src/object/obj_rpc.h

```diff
@@ -235,6 +238,22 @@ CRT_RPC_DECLARE(obj_punch, DAOS_ISEQ_OBJ_PUNCH, DAOS_OSEQ_OBJ_PUNCH)
 
 CRT_RPC_DECLARE(obj_query_key, DAOS_ISEQ_OBJ_QUERY_KEY, DAOS_OSEQ_OBJ_QUERY_KEY)
 
+#define DAOS_ISEQ_OBJ_SYNC	/* input fields */		 \
```

Comment:
(style)  Macros with complex values should be enclosed in parentheses

Reply:
Related macro definition is the general format for DAOS code.